unified-codes/apps/web/src/selectors/ExplorerSelectors.ts

```diff
@@ -45,6 +45,17 @@ const selectLabel = createSelector(selectFilterBy, (filterBy: EEntityField): str
   }
 });
 
+const selectPlaceholder = createSelector(selectFilterBy, (filterBy: EEntityField): string => {
+  switch (filterBy) {
+    case EEntityField.CODE:
+      return 'e.g. 4231441c';
+    case EEntityField.DESCRIPTION:
+      return 'e.g. amoxicillin, amox, *mox';
```

Comment:
what would be good is to explain that we default to 'starts with' - but I think there is enough instructional text already! I'd let people figure that out for themselves..

Reply:
Hmm. Yeah, I'm conflicted now. It _is_ a bit weird that we default to starts with although it also matches mSupply's weird behaviour...